src/Jira/Api/Client/CurlClient.php

```diff
@@ -78,6 +78,7 @@ public function sendRequest($method, $url, $data = array(), $endpoint, Authentic
         if ($method == "POST") {
             curl_setopt($curl, CURLOPT_POST, 1);
             if ($isFile) {
+                curl_setopt($curl, CURLOPT_SAFE_UPLOAD, false);
```

Comment:
This would result in an error prior PHP 5.5, where this option was added.

This was fixed in #52 .